.travis.yml

```diff
@@ -2,6 +2,7 @@ language: python
 python:
   - 2.6.9
   - 2.7.9
+  - 3.4
```

Comment:
3.4.3 wasn't available... I can pin to 3.4.2 if this build's green and you prefer.